src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,106 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 1px solid #f3f3f3;
+  border-radius: $border-radius;
+  padding: 32px 16px 17px;
+
+  &__img-wrapper {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__img {
+    width: 100%;
+    height: 100%;
+  }
+
+  &__content {
+    width: 166px;
+    color: $secondary-color;
+    font-size: $body-font-size;
+    line-height: $body-line-height;
+  }
+
+  &__title {
+    color: $main-color;
+    font-size: $accent-font-size;
+    font-weight: $medium-font-weigth;
+    line-height: $accent-lineheight;
+    padding: 40px 0 0;
+    margin: 0;
+  }
+
+  &__code {
+    margin-top: 4px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-top: 16px;
+  }
+  &__reviews-number {
+    margin: 0;
+    color: $main-color;
+  }
+  &__price-wrapper {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 24px;
+  }
+
+  &__price {
+    font-size: $accent-font-size;
+    margin: 0;
+  }
+
+  &__price-number {
+    color: $main-color;
+    font-size: 16px;
+    font-weight: $accent-font-weigth;
+    line-height: $accent-lineheight;
+    margin: 0;
+  }
+
+  &__btn {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    height: 40px;
+    margin-top: 16px;
+    box-sizing: border-box;
+    border-radius: $border-radius;
+    border: 1px solid transparent;
+    background: $accent-color;
+    color: #fff;
+    font-weight: $accent-font-weigth;
+    font-size: 14px;
+    line-height: 16px;
+    text-decoration: none;
+    text-transform: uppercase;
+
+    &:hover {
+      border-color: $accent-color;
+      color: $accent-color;
+      background: transparent;
+      transition: 0.5s ease-in-out;
```

Comment:
don't add transition on hover, add it under general selector, then it will work both ways